features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/MessageEventBubble.kt

```diff
@@ -88,7 +89,7 @@ fun MessageEventBubble(
         return if (state.isMine) {
             this
         } else {
-            offset(x = 20.dp)
+            offset(x = BUBBLE_OWN_OFFSET)
         }
```

Comment:
Is this constant named right? It seems like it's being used for items that are _not_ our own? :thinking:

Reply:
Ooops, nice catch.

Reply:
Fixed in https://github.com/vector-im/element-x-android/pull/573/commits/769fe337ebab6772fed88c48b385cd084848e53c.